packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -10,12 +10,16 @@ import {
   GEL_SPACING_DBL,
 } from '@bbc/gel-foundations/spacings';
 import Helmet from 'react-helmet';
-import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_B_MIN_WIDTH,
+  GEL_GROUP_CD_MIN_WIDTH,
+} from '@bbc/gel-foundations/breakpoints';
 import { getPica } from '@bbc/gel-foundations/typography';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 import { getSansRegular } from '@bbc/psammead-styles/font-styles';
 
-const MENU_ICON_SIDE_LENGTH = '2.75rem'; // 44px
+const MENU_BUTTON_TOP_BOTTOM_SPACING = 0.75; // 12px - must be the same as TOP_BOTTOM_SPACING in ../index.jsx
```

Comment:
hmm, could we export that value and use it in here?

Reply:
Yep, let's do that.